frodo.js

```diff
@@ -1,52 +1,50 @@
-/**
```

Comment:
SCRIPT-This new feature lacks unit tests. Could you add some to ensure everything works correctly?